.github/workflows/main.yml

```diff
@@ -24,7 +24,7 @@ env:
   # /dev.Dockerfile
   # /make/builder.Dockerfile
   # /.github/workflows/release.yml
-  GO_VERSION: 1.18.2
+  GO_VERSION: 1.19
```

Comment:
This has bitten us before as it would choose the latest version of the 1.19.x family. Can we pin this to 1.19.0 instead?